ui/dashboard/src/pages/project-details/environments/index.tsx

```diff
@@ -1,14 +1,24 @@
 import { useState } from 'react';
+import { Trans } from 'react-i18next';
+import { environmentArchive, environmentUnArchive } from '@api/environment';
```

Comment:
Same for the other places.

```suggestion
import { environmentArchive, environmentUnarchive } from '@api/environment';
```